templates/etc/init.d/elasticsearch.OpenBSD.erb

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# $OpenBSD: elasticsearch.rc,v 1.2 2015/07/15 23:34:49 ajacoutot Exp $
+
+daemon="/usr/local/elasticsearch/bin/elasticsearch"
+daemon_flags="-d -Des.config=/etc/elasticsearch/<%= @name %>/elasticsearch.yml -p <%= @pid_dir %>/elasticsearch-<%= @name %>.pid"
```

Comment:
I think i'm missing a few options here.
from the redhat init script:

```
-Des.default.path.home=$ES_HOME 
-Des.default.path.logs=$LOG_DIR 
-Des.default.path.data=$DATA_DIR 
-Des.default.path.work=$WORK_DIR 
-Des.default.path.conf=$CONF_DIR
```

Could you perhaps include the same options?